project.md

```diff
@@ -0,0 +1,96 @@
+# Rock-Paper-Scissors on the Micro:bit: A Semester Capstone Project
+
+## Abstract
+
+The goal of this semester capstone project is to demonstrate how the micro:bit—a pocket-sized computer designed for education—can be utilized to create an interactive and multiplayer game of Rock-Paper-Scissors. This paper highlights the technologies, inputs, outputs, and communication techniques used, to serve as a guide and inspiration for high school students embarking on their own semester projects.
+
+## Table of Contents
+
+1. Introduction
+2. Requirements and Pre-requisites
+3. Inputs and Outputs
+4. Communication Techniques
+5. Code Architecture
+6. Challenges and Lessons Learned
+7. Conclusion
+
+---
+
+## 1. Introduction
+
+The micro:bit is a versatile tool capable of a multitude of functions, ranging from simple displays to complex radio communications. We'll be taking advantage of its features to create a Rock-Paper-Scissors game that can be played in two modes—single-player against a computer and multiplayer via radio communication.
+
+---
+
+## 2. Requirements and Pre-requisites
+
+**Hardware Required:**
+- micro:bit (2 units for multiplayer)
+- USB cables for programming and power
+
+**Software Required:**
+- Python editor compatible with micro:bit (e.g., Mu)
+
+**Skills Required:**
+- Basic understanding of Python programming
+- Basic understanding of micro:bit's features
+
+---
+
+## 3. Inputs and Outputs
+
+### Inputs
+
+1. **Button A**: Cycles through the options (Rock, Paper, Scissors)
+2. **Button B**: Confirms the selection
+3. **Both Buttons**: Switches to multiplayer mode and selects host/client roles
+
+### Outputs
+
+1. **LED Display**: 
+    - Shows various emojis to indicate winner, loser, or tie
```

Comment:
does it have to be emojis or could it be anything that references towards the action